lib/db/schema/patch-022-023.sql

```diff
@@ -0,0 +1,133 @@
+-- This migration:
+--
+-- * Changes the type of the devices.callbackPublicKey column
+-- * Add a new devices.callbackAuthKey column
+-- * Updates the accountDevices stored procedure with the new column
+-- * Alters three stored procedures, createDevice, updateDevice and
+--   sessionWithDevice to work with the new columns
+
+ALTER TABLE devices
+MODIFY COLUMN callbackPublicKey VARCHAR(88),
+ADD COLUMN callbackAuthKey VARCHAR(24);
```

Comment:
Please try adding the following declarations on to the end of this `ALTER TABLE`:

````
ALTER TABLE devices <...existing declarations>, ALGORITHM = INPLACE, LOCK = NONE;
```

This will cause the alter to fail if it can't be applied in an online manner.  If it can't, we're going to have trouble deploying it in production, so best to put the check in early.